index.html

```diff
@@ -0,0 +1,177 @@
+    <title>Save the plannet conference</title>
+    <link rel="stylesheet" href="https://use.typekit.net/kix3hqc.css">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav>
+      <ul class="nav-mobile2">
+        <li><a href="About.html" class="links">About</a></li>
+        <li><a href="#programs" class="links">Programs</a></li>
+        <li><a href="#speakers" class="links">Speakers</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="lines"></span>
+        <span class="lines"></span>
+        <span class="lines"></span>
+      </div>
+      <ul class="nav-desktop">
+        <li class="abt"><a href="About.html" class="desktop-items">About</a></li>
+        <li><a href="#about" class="desktop-items">Program</a></li>
+        <li><a href="#" class="desktop-items">Join</a></li>
+        <li><a href="#" class="desktop-items">Donate</a></li>
+        <li><a href="#about" class="desktop-items">Chat</a></li>
+        <li><a href="#" class="desktop-items">Subscribe</a></li>
+      </ul>
+    </nav>
+    <div class="my-logo">
+      <img src="images/copy4.PNG" alt=""/>
+      <div class="log-2">
+      <p>Global footprint</p>
+      <p class="pp"> network</p>
+    </div>
+    </div>
+    </header>
+    
+    <main class="main-home">
+      <div class="sub-main">
+        
+      <section class="home-1">
+       <div class="part1">
+        <div class="tittle">
+          <h5>"All the human race"</h5>
+          <h1>SAVE THE PLANNET INTERNATIONAL CONFERENCE 2O24</h1>
+        </div>
+        <div class="details">
+          <p>
+            Intergovernmental Panel on Climate Change (IPCC) on
+            20 March. “This Synthesis Report underscores the urgency of taking
+            more ambitious action 
+        </div>
+        <div class="dates">
+          <h2>2024.12.17(FRI) ~ 20(MON)</h2>
+          <p>@Kampala serena hotel, kampala </p>
+        </div>
+      </div>
+    </div>
+        <div class="programs" id="programs">
+          <div class="agenda">
+          <h2>Agenda</h2> 
+          <hr> 
+        </div>
+        <section class="programs-container">
+        <div class="all-programs">
+          <div class="program1">
+            <div>
+              <img src="images/icon1.png" alt="" />
+            </div>
+            <div>
+              <h4>Remark</h4>
+            </div>
+            <div>
+              <p>Key research outcomes from scients around globe
+              </p>
+          </div>          
+          </div>       
+          <div class="program2">
+            <div>
+            <img src="images/icons8-user-48.png" alt="" />
+          </div>
+          <div>
+            <h4>cc Forum.</h4>
+          </div>
+          <div>
+            <p>Discussions of way forward from different delegates 
+            </p>
+          </div>
+          </div>
+          <div class="program3">
```

Comment:
- [ ] Kindly ensure that each of these cards are wrapped in `article` tags as each of the cards are articles